ui/src/components/groups/GroupTable.vue

```diff
@@ -95,7 +95,9 @@
       must-sort
       :item-class="itemClass"
       class="elevation-1"
-      :footer-props='{itemsPerPageText: $t("$vuetify.dataTable.rowsPerPageText")}'
+      :footer-props="{
```

Comment:
Do we still need `:footer-props` with Qiang's I18N update?

Reply:
I will remove them now!